browser/net/brave_request_handler.cc

```diff
@@ -68,8 +69,10 @@ void BraveRequestHandler::SetupCallbacks() {
   callback = base::BindRepeating(brave::OnBeforeURLRequest_AdBlockTPPreWork);
   before_url_request_callbacks_.push_back(callback);
 
-  callback = base::BindRepeating(brave::OnBeforeURLRequest_HttpsePreFileWork);
-  before_url_request_callbacks_.push_back(callback);
+  if (!base::FeatureList::IsEnabled(net::features::kBraveHttpsByDefault)) {
+    callback = base::BindRepeating(brave::OnBeforeURLRequest_HttpsePreFileWork);
```

Comment:
shouldn't we also hide the corresponding shields UI?

Reply:
Yes -- the old shields UI (for httpse) is already hidden when this flag is enabled, by 98b811e1ae6b74f3be8456c6008f791ea2c41b9b and f84a6b0967686ed516724fa6bb1fef290d0ac964